intellij-ide.md

```diff
@@ -188,8 +188,22 @@ types of code changes cannot be hot reloaded though:
 For these changes you can fully restart your application, without having to end
 your debugging session:
 
-1. In the Debug panel at the bottom, click the **Full Application Restart button**
-(shown with an icon containing a circular arrow).
+1. Don't click the Stop button; simply re-click the Run button (if in a run
+session) or Debug button (if in a debug session).
```

Comment:
Does the current explanation of the differences (~ 5 lines above your comment) not explain reload vs restart?